src/GraphNavigator/SerializationGraphNavigator.php

```diff
@@ -234,6 +235,10 @@ public function accept($data, ?array $type = null)
                     throw new ExcludedClassException();
                 }
 
+                if (!is_object($data)) {
+                    throw new InvalidArgumentException('Value at '.$this->context->getPath().' is expected to be an object of class '.$type['name'].' but is of type '.gettype($data));
```

Comment:
getPath only gives the model class but not the current property. do we know the name of the property we are currently looking at?